FilePermissionTest fails on release_build.sh

Logs:
```
[ERROR] Tests run: 8, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 0.009 s <<< FAILURE! - in com.google.cloud.tools.io.FilePermissionsTest
[ERROR] testDirectoryCannotBeCreatedDueToUnwritableParent(com.google.cloud.tools.io.FilePermissionsTest)  Time elapsed: 0.006 s  <<< FAILURE!
java.lang.AssertionError: Can create directory in non-writable parent
	at com.google.cloud.tools.io.FilePermissionsTest.testDirectoryCannotBeCreatedDueToUnwritableParent(FilePermissionsTest.java:102)
```

It fails on the `release_build.sh` script, but passes in the CIs. Not sure why...

--- a/kokoro/release_build.sh
+++ b/kokoro/release_build.sh
@@ -9,7 +9,7 @@ sudo -E /opt/google-cloud-sdk/bin/gcloud components update -q
 sudo -E /opt/google-cloud-sdk/bin/gcloud components install app-engine-java -q
 
 cd github/appengine-plugins-core
-mvn -Prelease -B -U verify
+mvn -Prelease -B -U verify -Dtest=!FilePermissionsTest
 
 # copy pom with the name expected in the Maven repository
 ARTIFACT_ID=$(mvn -B help:evaluate -Dexpression=project.artifactId 2>/dev/null | grep -v "^\[")

